docs/spec/trustbloc-did-method.md

```diff
@@ -95,12 +95,18 @@ Example of the format of the configuration data wrapped within the JWS:
         {
             "domain": "stakeholder.one",
             "did": "[stakeholder one's DID]",
-            "key": "[stakeholder one's did:key verification key]"
+            "public_key": {
+                "id": "[stakeholder one's verification public key ID]",
+                "jwk": {stakeholder one's verification public key in JWK format},
+            }
         },
         {
             "domain": "stakeholder.two",
             "did": "[stakeholder two's DID]",
-            "key": "[stakeholder two's did:key verification key]"
+            "public_key": {
+                "id": "[stakeholder two's verification public key ID]",
```

Comment:
it's still not clear to me how this ID correlates to the publicKey entry in the did document. Is it a DID URL?

Reply:
Updated to a DID URL. We could also potentially make it a relative DID URL.